### PR TITLE
solution

### PR DESCRIPTION
**Pull request by gnosis-frog**
[DEMO LINK](https://gnosis-frog.github.io/react_list-of-goods/)

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -14,11 +14,106 @@ const goodsFromServer: string[] = [
...
+      <div className="App">
+        {!isListDisplayed && (
+          <button
+            className="button is-primary"
+            type="button"
+            onClick={() => {
+              this.setState({
+                isListDisplayed: true,
+              });
+            }}
```
Comment (OleksandrOstrovskyi) [inline src/App.tsx:64]: Варто винести окремим методом

**Review by OleksandrOstrovskyi — APPROVED** [review]
Виглядає гарно. Залишив комент-рекомендацію

File: src/App.scss [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+.button {
+  margin: 20px 10px 10px 0;
+}
+
+.box {
+  width: 490px;
+
+  cursor: pointer;
+  transition: background-color, transform 0.3s;
```
Comment (Olexandr6) [inline src/App.scss:13]: If you want to set duration for certain CSS properties, you should write duration time for each one.
```suggestion
  transition: background-color 0.3s, transform 0.3s;
```

Comment (Olexandr6) [reply src/App.scss:13]: In your case `background-color` transition duration will be 0sec by default

File: src/App.tsx [hunk 3]
```diff
@@ -14,11 +14,108 @@ const goodsFromServer: string[] = [
...
+
+  sortAplphabetically = () => {
+    this.setState(state => ({
+      goods: [...state.goods].sort((goodA, goodB) => (
+        goodA.localeCompare(goodB)
+      )),
+    }));
+  };
+
+  reset = () => {
```
Comment (Olexandr6) [inline src/App.tsx:37]: reset what??? Rename this function

**Review by Olexandr6 — APPROVED** [review]
Good job 👍 But some small things should be fixed

**Pull request closed without merging**